Tests/UnitTests/Networking/HTTPClientTests.swift

```diff
@@ -915,7 +956,7 @@ final class HTTPClientTests: BaseHTTPClientTests<MockETagManager> {
 
         let request = HTTPRequest(method: .post([:]), path: .mockPath)
 
-        var headerPresent = false
+        var headerPresent = true
```

Comment:
This test is testing for `false` so changed the `headerPresent` to default to `true` so it will default fail

Reply:
👍🏻  You can leave this test as is since you didn't add it, and this does prevent a false positive